geometry_msgs/PointStamped poi  # point of interest for grasping
sensor_msgs/PointCloud2 cloud   # point cloud over which to calculate grasps